static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import fetch from 'lib/fetch';
+
+const isCheckbox = element => element.type === 'checkbox';
+const isValid = element => element.name !== '';
```

Comment:
I'm no expert here so I have no idea whether we should worry about `null` or `undefined`. If we do then I think the expression `element.name > ''` may serve the purpose.